web/src/features/charts/PriceChart.tsx

```diff
@@ -42,7 +44,11 @@ function PriceChart({ datetimes, timeAverage }: PriceChartProps) {
     }));
   }
 
-  if (!chartData[0]?.layerData?.price) {
+  if (
+    chartData[0]?.layerData?.price === undefined ||
+    Number.isNaN(chartData[0]?.layerData?.price) ||
+    chartData[0]?.layerData?.price === null
+  ) {
```

Comment:
This had to be changed since the previous implementation would result in not showing price data if the first price entry was 0.